lib/generation/generators/util/pb_input_formatter.dart

```diff
@@ -15,7 +15,9 @@ class PBInputFormatter {
         spaceToUnderscore: spaceToUnderscore, destroyDigits: destroyDigits);
 
     (isTitle)
-        ? result = result.replaceRange(0, 1, result[0].toUpperCase())
+        ? result = result.camelCase
```

Comment:
Maybe we can use the `import 'package:recase/recase.dart';` imported to just comvert the title into `PascalCase` like so 
```dart
result = result.PascalCase;
```
or we can use any of the other options "Title Case" or "snake_case". 🤔

Reply:
I will test the outcome of this, thanks!